docs/cloud/secret-management.md

```diff
@@ -0,0 +1,93 @@
+# Secret management
+
+:::caution Under Construction
+
+This section is under construction.
```

Comment:
You've already mentioned with the `caution` that it's under construction.

Reply:
Removed the line.